src/applications/disability-benefits/526EZ/actions/index.js

```diff
@@ -1,64 +1,156 @@
-/* eslint-disable no-unused-vars */
 import { apiRequest } from '../../../../platform/utilities/api';
-import { fetchInProgressForm } from '../../../common/schemaform/save-in-progress/actions';
 
-export const ITFStatuses = Object.freeze({
+import existingITFData from '../tests/itfData';
+
+import { getLatestTimestamp } from '../helpers';
+
+export const PRESTART_STATUS_SET = 'PRESTART_STATUS_SET';
+export const PRESTART_DATA_SET = 'PRESTART_DATA_SET';
+export const PRESTART_RESET = 'PRESTART_STATUS_RESET';
+export const PRESTART_DISPLAY_RESET = 'PRESTART_DISPLAY_RESET';
```

Comment:
I considered using ITF here but decided against it because it's an acronym that might not be as easy to maintain/conceptualize down the road, also we are actually retrieving multiple ITF, an array of them if the user has any, and then later, potentially creating one. It seems helpful to use prestart here to express that it is the gating check that this refers to, and not one of the various individual retrieved or created ITFs.